doc/content/reference/components/network-operations-center.md

```diff
@@ -47,32 +51,124 @@ The following is a brief description of each column in the table:
+
+The following is a brief description of each column in the table:
+
+- **Application ID**: displays application IDs of all the applications on the network.
+- **Active Devices**: the number of devices in the application which have uplinked in the **last hour**.
+- **Average Packet Error Rate**: this is calculated by comparing the number of seen frames to the total number of possible frames.
+
+### End Device Overview
+
+The end device overview consists of the following panels:
+
+**End devices active** - a bar chart that visualizes the number of devices that have connected to the network in the selected period, either in the last hour, week, or day.
+
+**End device statistics** - a table layout that displays the **End Device ID**, **Application ID**, **Last Seen**, and **Last Packet Error Rate** of all the devices on the network in the selected period.
+
+The following is a brief description of each column in the table:
+
+- **End Device ID**: displays IDs of all the end devices on the network.
+- **Application ID**: displays the application ID for each device.
+- **Last Seen**: displays when the device last uplinked.
+- **Last Packet Error Rate**: .
+
 ## Gateway Details Dashboard
 The Gateway Details dashboard visualizes detailed insight into gateway statistics in the selected time period.
 
 {{< figure src="../gateway-details-dashboard.png" alt="gateway details dashboard" >}}
 
 It consists of the following panels:
 
-**Uptime** - the gauge chart that visualizes the gateway uptime, i.e. the fraction of the time in which the gateway has been connected to the Network Server.
+**Uptime** - a gauge chart that visualizes the amount of time the gateway has been connected to the Network Server, represented as a percentage out of 100.
 
-{{< figure src="../uptime.png" alt="uptime" >}}
+{{< figure src="../gateway-uptime.png" alt="uptime" >}}
 
-**Lowest RSSI (LoRa)** - the graph that visualizes data about the lowest Received Signal Strength Indicator (RSSI) in dBm for each channel. 
+**Lowest RSSI (LoRa)** - a graph that visualizes data about the lowest Received Signal Strength Indicator (RSSI) in dBm for each channel. 
 
-{{< figure src="../rssi.png" alt="lowest rssi" >}}
+{{< figure src="../gateway-rssi.png" alt="lowest rssi" >}}
 
 **Traffic** - a time series graph that visualizes the uplink and downlink rates in packets per second.
 
-{{< figure src="../traffic.png" alt="traffic" >}}
+{{< figure src="../gateway-traffic.png" alt="traffic" >}}
 
 **Packets per channel** - a heat map that visualizes the number of uplink data packets for each channel.
 
-{{< figure src="../packets-per-channel.png" alt="packets per channel" >}}
+{{< figure src="../gateway-packets-per-channel.png" alt="packets per channel" >}}
 
-**Channel utilization** - a graph that visualizes the percentage of total utilization on each channel over a time interval.
+**Channel utilization** - a chart displaying the percentage of packets received on each channel.
```

Comment:
Are you sure about this? Because I think utilization means which % of channel is occupied, and not how many packets are received

Reply:
It's not entirely the same thing, because utilization takes the packet duration into account. If you have two identical packets, but one is transmitted at SF7 and one at SF8, the SF8 one has double duration and in turn double utilization.

Reply:
Fixed. 
The graph uses 15-minute time slots to calculate channel utilization, right?
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/36432168/a2195927-997a-4342-9b94-480620daa45d)